pkg/metrics/metrics.jsx

```diff
@@ -641,6 +646,39 @@ class CurrentMetrics extends React.Component {
             cpu_label = this.state.cpuUsed + '%';
         }
 
+        let disksUsage = [];
+        if (this.disksNames.length > 0 && this.samples[14] && this.samples[15]) {
+            disksUsage = this.disksNames.map((name, i) => [
+                name,
+                this.samples[14][i] >= 1 ? cockpit.format_bytes_per_sec(this.samples[14][i]) : "0",
+                this.samples[15][i] >= 1 ? cockpit.format_bytes_per_sec(this.samples[15][i]) : "0",
+            ]);
+            disksUsage = disksUsage.filter(disk => !disk[0].match(/^zram[0-9]+$/));
+        }
+
+        let allDisks = null;
+        if (disksUsage.length > 1) {
+            const disksTableContent = (
+                <Table
+                    variant={TableVariant.compact}
+                    gridBreakPoint={TableGridBreakpoint.gridLg}
+                    borders={false}
+                    aria-label={ _("Disks usage") }
+                    cells={ [{ title: _("Device"), transforms: [fitContent] }, _("Read"), _("Write")] }
+                    rows={disksUsage}
+                    rowWrapper={ props => <RowWrapper device-name={ props.row[0] } {...props} /> }>
+                    <TableHeader />
+                    <TableBody className="pf-m-tabular-nums disks-nowrap" />
+                </Table>
+            );
+
+            allDisks = (
+                <Popover minWidth={0} aria-label={ _("View all disks") } bodyContent={disksTableContent}>
```

Comment:
Do we have any chance of only creating `disksTableContent` when it's actually visible? Most of the time this would be generated in vain. I didn't see anything in https://www.patternfly.org/v4/components/popover/ which could help here; bodyContent *can* take a function, but only to pass a "hide" callback, so that doesn't help us here.

Just thinking aloud, probably not actionable.